enterprise/server/remote_execution/cgroup/cgroup_test.go

```diff
@@ -4,13 +4,83 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/remote_execution/block_io"
+	"github.com/buildbuddy-io/buildbuddy/server/util/proto"
 	"github.com/google/go-cmp/cmp"
 	"github.com/stretchr/testify/require"
 	"google.golang.org/protobuf/testing/protocmp"
 
 	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
+	scpb "github.com/buildbuddy-io/buildbuddy/proto/scheduler"
 )
 
+func TestSettingsMap(t *testing.T) {
+	for _, test := range []struct {
+		name     string
+		settings *scpb.CgroupSettings
+		expected map[string]string
+	}{
+		{
+			name:     "nil",
+			settings: nil,
+			expected: map[string]string{},
+		},
+		{
+			name:     "all fields unset",
+			settings: nil,
+			expected: map[string]string{},
+		},
```

Comment:
these 2 tests look the same

Reply:
woops, meant to use an empty proto here.